lib/models/application.coffee

```diff
@@ -446,6 +448,108 @@ getApplicationModel = (deps, opts) ->
 		.asCallback(callback)
 
 	###*
+	# @summary Purge
+	# @name purge
+	# @public
+	# @function
+	# @memberof resin.models.application
+	#
+	# @param {Number} appId - application id
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.application.purge(123, 456);
+	#
+	# @example
+	# resin.models.application.purge(123);
+	#
+	###
+	exports.purge = (appId) ->
+		return request.send
+			method: 'POST'
+			url: '/supervisor/v1/purge'
+			baseUrl: apiUrl
+			body:
+				appId: appId
+				data:
+					appId: "#{appId}"
+		.catch (err) ->
+			if err.statusCode == LOCKED_STATUS_CODE
+				throw new errors.ResinSupervisorLockedError()
+
+			throw err
+
+	###*
+	# @summary Shutdown
+	# @name shutdown
+	# @public
+	# @function
+	# @memberof resin.models.application
+	#
+	# @param {Number} appId - application id
+	# @param {Boolean} force (optional)
```

Comment:
Can you change this to an options object so that it matches `resin.models.device.shutdown()`